freight/notifiers/datadog.py

```diff
@@ -0,0 +1,51 @@
+from __future__ import absolute_import
+
+__all__ = ['DatadogNotifier']
+
+from freight import http
+from freight.models import App, TaskStatus
+
+from .base import Notifier, NotifierEvent
+
+
+class DatadogNotifier(Notifier):
+    def get_options(self):
+        return {
+            'webhook_url': {'required': True},
+        }
+
+    def should_send_deploy(self, deploy, task, config, event):
+        if event == NotifierEvent.TASK_STARTED:
+            return True
+
+        if event == NotifierEvent.TASK_FINISHED and task.status == TaskStatus.finished:
+            return True
+
+        return False
+
+    def send_deploy(self, deploy, task, config, event):
+        webhook_url = config['webhook_url']
```

Comment:
maybe call it `event_webhook_url` or something and put the datadog api doc link in a comment/docstring somewhere in here? Just so we have a reference for what this payload means to datadog.

Reply:
done!